docs/source/conf.py

```diff
@@ -92,6 +92,7 @@ def setup(app):
     "vllm._C",
     "PIL",
     "numpy",
+    'triton'
```

Comment:
You forgot a comma here. This might have messed up the docs

Reply:
Good catch (though I'm not sure about why the docs still build, both locally and remotely), will make a quick PR to fix